src/js/component/parkingMap/index.js

```diff
@@ -0,0 +1,62 @@
+import axios from 'axios';
+import React, { useEffect, useState } from 'react';
+import { useLocation } from 'react-router-dom';
+import { withStyles } from '@mui/styles';
+import styles from './styles';
+import { useSelector } from 'react-redux';
+import { getRefreshedPageData } from '../../store/common/selectors';
+import { get } from '../../axiosUtils/appUtils';
+
+const ParkingMap = (props) => {
+  const { classes } = props;
+  const { state } = useLocation();
+  const [dataSet, setDataset] = useState([]);
+  const [parkingStatus, setParkingStatus] = useState({});
+  const refreshFlag = useSelector((state) => getRefreshedPageData(state));
+
+  useEffect(() => {
+    axios.get('/db/parkingMapData.json').then((response) => {
```

Comment:
We should have dispatch things from this layer to action and waha pe we can call our apputil.js which already has axios layer right ?

Reply:
I am sure there is some reason aap ne ye keya hoga ?

Reply:
we discussed